src/en/holymanga/src/eu/kanade/tachiyomi/extension/en/holymanga/HolyManga.kt

```diff
@@ -1,13 +1,35 @@
 package eu.kanade.tachiyomi.extension.en.holymanga
 
 import eu.kanade.tachiyomi.multisrc.fmreader.FMReader
+import eu.kanade.tachiyomi.source.model.SChapter
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
 
 class HolyManga : FMReader(
     "HolyManga",
     "https://w34.holymanga.net",
     "en",
+    SimpleDateFormat("yyyy-MM-dd", Locale.ENGLISH),
 ) {
     override val versionId = 2
 
     override val chapterUrlSelector = ""
+
+    override fun chapterFromElement(element: Element, mangaTitle: String): SChapter {
+        return SChapter.create().apply {
+            if (chapterUrlSelector != "") {
+                element.select(chapterUrlSelector).first()!!.let {
+                    setUrlWithoutDomain(it.attr("abs:href"))
+                    name = it.text().substringAfter("$mangaTitle ")
+                }
+            } else {
+                element.let {
+                    setUrlWithoutDomain(it.attr("abs:href"))
+                    name = element.attr(chapterNameAttrSelector).substringAfter("$mangaTitle ")
+                }
+            }
+            date_upload = element.select(chapterTimeSelector).let { if (it.hasText()) parseAbsoluteDate(it.text()) else 0 }
+        }
+    }
```

Comment:
Call the super method?

https://github.com/keiyoushi/extensions-source/blob/5988c02c57f04d356a2efbe25d0a1e3d25bc02d9/src/vi/doctruyen3q/src/eu/kanade/tachiyomi/extension/vi/doctruyen3q/DocTruyen3Q.kt#L89-L93

https://github.com/keiyoushi/extensions-source/blob/5988c02c57f04d356a2efbe25d0a1e3d25bc02d9/src/en/xoxocomics/src/eu/kanade/tachiyomi/extension/en/xoxocomics/XoxoComics.kt#L106-L110